Tests/base/NSFileHandle/singleton.m

```diff
@@ -21,6 +21,10 @@ int main(void)
   PASS_EXCEPTION([a release], NSGenericException, "Cannot dealloc stdin");
   PASS_EXCEPTION([b release], NSGenericException, "Cannot dealloc stdout");
   PASS_EXCEPTION([c release], NSGenericException, "Cannot dealloc stderr");
+  // The following are expected to fail with an ARC-supporting runtime.  ARC
+  // doesn't allow resurrection and the objc_retain() function will check for
+  // this so the [self retain] in GSFileHandle's -dealloc will not actually
+  // resurrect the object.
```

Comment:
Ah, this is why TravisCI is erroring out.

If this is true, should we not omit them with an ARC-supporting runtime? Or, should we not replace them with the opposite test?

I don't think we should intentionally break tests, especially if we agree that use of CI is useful. What do you think?

Reply:
Yes, I need to fix the tests before committing.  Simply not running these ones on an ARC runtime is probably fine.